community/weechat/depends

```diff
@@ -2,3 +2,4 @@ cmake     make
 libgcrypt
 libressl
 zlib
+gnutls
```

Comment:
Sort this alphabetically.

Reply:
Sorry about that. depends has been sorted.